server/utils/premises/occupancy.test.ts

```diff
@@ -128,26 +128,87 @@ describe('apOccupancy utils', () => {
   })
 
   describe('daySummaryRows', () => {
-    it('should generate a list of day summary rows', () => {
-      const capacityForDay = cas1PremiseCapacityForDayFactory.build({
+    const emptyRow = {
+      key: { html: `<div class="govuk-!-static-padding-top-5"></div>` },
+      value: null as { text: string },
+    }
+    const daySummary = cas1PremisesDaySummaryFactory.build({
+      capacity: cas1PremiseCapacityForDayFactory.build({
+        date: '2025-02-02',
         totalBedCount: 20,
         availableBedCount: 18,
-        bookingCount: 6,
-      })
-      const daySummary = cas1PremisesDaySummaryFactory.build({
-        capacity: capacityForDay,
-      })
+        bookingCount: 21,
+        characteristicAvailability: [
+          premiseCharacteristicAvailability.build({
+            characteristic: 'hasEnSuite',
+            availableBedsCount: 1,
+            bookingsCount: 2,
+          }),
+          premiseCharacteristicAvailability.build({
+            characteristic: 'isWheelchairDesignated',
+            availableBedsCount: 2,
+            bookingsCount: 1,
+          }),
+          premiseCharacteristicAvailability.build({
+            characteristic: 'isStepFreeDesignated',
+            availableBedsCount: 1,
+            bookingsCount: 1,
+          }),
+        ],
+      }),
+    })
+    it('should generate a list of day summary rows when no criteria are provided', () => {
       const expected = [
         { key: { text: 'Capacity' }, value: { text: '20' } },
-        { key: { text: 'Booked spaces' }, value: { text: '6' } },
+        { key: { text: 'Booked spaces' }, value: { text: '21' } },
         { key: { text: 'Out of service beds' }, value: { text: '2' } },
-        { key: { text: 'Available spaces' }, value: { text: '12' } },
+        { key: { text: 'Available spaces' }, value: { text: '-3' } },
       ]
 
       expect(daySummaryRows(daySummary)).toEqual({
         rows: expected,
       })
     })
+
+    it('should generate a list of day summary rows including criteria', () => {
+      const expected = [
+        { key: { text: 'Capacity' }, value: { text: '20' } },
+        { key: { text: 'Booked spaces' }, value: { text: '21' } },
+        { key: { text: 'Out of service beds' }, value: { text: '2' } },
+        { key: { text: 'Available spaces' }, value: { text: '-3' } },
+        emptyRow,
```

Comment:
This isn't great semantically -- I wonder what a screen reader would make of the resulting markup. Could this be split into 2 summary lists?

Reply:
Actually, I don't think the blank line is needed at all with the POC layout as I don't think there is any useful info in the old table.
I'd rather not spend time improving this blank line thing until we know that it's needed.

Reply:
Fair enough, let's go ahead with this!